src/webui/api/searchcontroller.cpp

```diff
@@ -301,7 +301,7 @@ QString SearchController::generateSearchId(const QString &sessionId) const
  *   - "siteUrl"
  *   - "descrLink"
  */
-QJsonObject SearchController::getResults(const QList<SearchResult> &searchResults, const bool isSearchActive, const int totalResults) const
+QJsonObject SearchController::getResults(const QList<SearchResult> &searchResults, bool isSearchActive, int totalResults) const
```

Comment:
I said:
>Please don't use `const` in function **declaration** with non-pointer/non-reference types. It's useless and confusing. Leave it in function definition only.

So you can leave it here.

Reply:
I wasn't aware this could differ between the function declaration and definition.